homeassistant/components/http/__init__.py

```diff
@@ -264,6 +273,28 @@ async def start_server(*_: Any) -> None:
         local_ip, host, server_port, ssl_certificate is not None
     )
 
+    @callback
```

Comment:
I had to add callback to make it run in the event loop (this is my first time using the annotation)